src/pages/New.js

```diff
@@ -22,23 +22,29 @@ const AddCircleIcon = styled(Add)`
 
 export default function New() {
   function handleAccept() {
-    postRestaurant({
+    return postRestaurant({
       imgSrc:
         "https://static.lieferando.de/images/restaurants/de/OPROO0P/logo_465x320.png",
-      title: "Pasta Pronto",
+      title: `${Input}`,
       categories: ["pizza", "salad", "pasta"],
       distance: 6,
       rating: 3.1,
       description: "Eat italian"
     });
   }
 
+  const [Input, getInput] = React.useState("");
```

Comment:
Good job! It's great, that you added a default value.
But we give variables (const defines two variables here) camelCase names (except for Components).
Please rename `Input` to `input` in this file.